Google.Api.Generator.Utils/Formatting/WhitespaceFormatter.cs

```diff
@@ -78,6 +78,13 @@ private IDisposable WithIndent(bool withIndent = true)
         private SyntaxTriviaList FormatXmlDoc(SyntaxTriviaList trivList) =>
             XmlDocSplitter.Split(_indentTrivia, _maxLineLength, trivList);
 
+        private IEnumerable<SyntaxTrivia> AdjustTrivia(SyntaxTrivia triv) => triv switch
```

Comment:
I think this is work a comment. I can't immediately guess what this is trying to do.

Reply:
Changed the name of the method and added comments.